index.html

```diff
@@ -201,5 +201,19 @@ <h2 class="aheading">About Myself</h2>
   </div>
         </div>
       </section>
+      <div >
+      <footer class="contact-form"> 
+      <form action="https://formspree.io/f/{form_id}" method="post">
+        <ul class="form_items">
+<li class="form_items_1"> <h1 class="contact-title">Contact me</h1></li>
+<li class="form_items_1"><p class="contact-text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p></li>
+<li class="form_items_1"><input class="formtext" type="text" name="name" placeholder="Burak" maxlength="30" required></li>   
+<li class="form_items_1"><input class="formemail" type="email" name="email" placeholder="r.brk.otman@gmail.com" required></li>
+<li class="form_items_1"><textarea class="formtextarea" placeholder="Write your message here" maxlength="500" required></textarea></li>    
+<button type="submit" class="btnform">Get It Touch</button>
```

Comment:
- [ ] Since you are using `ul` and `li` to structure the form, I highly suggest you wrap this button element in a `li` tag to make it semantically more correct. You can check this [documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul) on MDN to get more information about what elements are permitted inside a `ul` tag. 🥸️

**Example**

```
<ul>
  ...
  <li><button type="submit" class="btnform">Get It Touch</button></li>
</ul>
```